app/upgrades/v15/upgrade_test.go

```diff
@@ -76,13 +76,19 @@ func (suite *UpgradeTestSuite) TestRegisterOsmoIonMetadata() {
 	ctx := suite.Ctx
 	bankKeeper := suite.App.BankKeeper
 
+	uosmoMetadata, found := suite.App.BankKeeper.GetDenomMetaData(ctx, "uosmo")
+	suite.Require().True(!found)
+
+	uionMetadata, found := suite.App.BankKeeper.GetDenomMetaData(ctx, "uion")
+	suite.Require().True(!found)
```

Comment:
```suggestion
	suite.Require().False(found)

	uionMetadata, found := suite.App.BankKeeper.GetDenomMetaData(ctx, "uion")
	suite.Require().False(found)
```
Annoying nit, sorry :sweat_smile: